Modules/_decimal/tests/runall-memorydebugger.sh

```diff
@@ -34,16 +34,16 @@ case $@ in
 esac
 
 # gmake required
-GMAKE=`which gmake`
```

Comment:
The filename ends in `.sh`, which is Bourne shell. `man dash` (Debian's Bourne compatible shell), does not mention "legacy" or "deprecation" of backticks.